src/anemoi/models/models/hierarchical.py

```diff
@@ -0,0 +1,347 @@
+# (C) Copyright 2024 ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+#
+
```

Comment:
Can the copyright statement be updated in accordance with https://github.com/ecmwf/anemoi-models/pull/67?

Reply:
Done ✔️